src/content/platform-integration/web/faq.md

```diff
@@ -120,18 +120,109 @@ See [Preparing a web app for release][].
 
 Not currently.
 
+### Why doesn't my app update immediately after it's deployed?
+
+You may need to configure the Cache-Control header returned by your web server. 
+For example, if this header is set to 3600, then the browser
+and CDN will cache the asset for 1 hour, and your users may see an out-of-date
+version of your app up to 1 hour after you deploy a new version. For
+more information about caching on the web,
+see [this web.dev article][http-cache] on HTTP caching.
+
+It is a good idea to be aware of this behavior to avoid breaking users.
+After you deploy your app, users may see a 
+cached version of your app (cached by the browser or CDN)
+for the duration defined by your cache headers.
+This can lead to users using a version of your app that
+are incompatible with changes that have been deployed to backend services.
+
+### How do I bust the web cache after each deployment?
```

Comment:
```suggestion
### How do I "bust the web cache" after a deployment and force an app download?
```

OK, I was going to reword "bust the web cache" but I googled and it's definitely a thing. But I'm adding a tad of an explanation to the anchor title.

Reply:
I think I'm going to change this to "clear the web cache" and remove the quotes.